src/test/app/claims/responseModelConverter.ts

```diff
@@ -279,7 +279,7 @@ describe('ResponseModelConverter', () => {
 
         it('should convert partial admission paid immediately', () => {
           const responseDraft = prepareResponseDraft(partialAdmissionWithImmediatePaymentDraft, individualDetails)
-          const responseData = preparePartialResponseData({ ...partialAdmissionWithImmediatePaymentData, ...directionsQuestionnaireResponseData }, individual)
+          const responseData = preparePartialResponseData({ ...partialAdmissionWithImmediatePaymentData(), ...directionsQuestionnaireResponseData }, individual)
```

Comment:
Is it worth mixing up different defendants, e.g company as a defendant.

Reply:
Could do! There are certainly lots of cases where our unit tests and route tests are picking one (normally individual v individual) and assuming that pass/fail applies to all combinations. We could raise a tech-debt to address it.